pkg/cmd/server/api/types.go

```diff
@@ -296,19 +296,19 @@ type MasterConfig struct {
 
 // JenkinsPipelineConfig holds configuration for the Jenkins pipeline strategy
 type JenkinsPipelineConfig struct {
```

Comment:
We can, although if you want to temporarily disable it do you also want to
wipe out the config?

On Apr 29, 2016, at 9:58 PM, Jordan Liggitt <notifications@github.com>
wrote:

In pkg/cmd/server/api/types.go
<https://github.com/openshift/origin/pull/8693#discussion_r61663013>:

> @@ -296,19 +296,19 @@ type MasterConfig struct {
>
>  // JenkinsPipelineConfig holds configuration for the Jenkins pipeline strategy
>  type JenkinsPipelineConfig struct {

for other optional portions of the config, we made the parent struct a
pointer... was there a reason not to do that here?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/8693/files/05c0b1f510a5ac62d6e5710d3a4101393085974a..e6bcd3d10d02c4ce06374f1bb2dafb2c89c901f1#r61663013>